app/components/pool/action/add/pool-create-basic-dialog.html

```diff
@@ -98,7 +98,9 @@
                 <div no-value-title>Add a start task</div>
                 <div value-title>{{startTask?.commandLine}}</div>
                 <div nested-form>
-                    <bl-start-task-picker [formControl]="startTaskPicker.nestedValue" [userAccounts]="form.value.userAccounts"></bl-start-task-picker>
+                    <bl-start-task-picker [formControl]="startTaskPicker.nestedValue"
+                                          [userAccounts]="form.value.userAccounts"
+                                          [virtualMachineConfiguration]="form.value.os.virtualMachineConfiguration"></bl-start-task-picker>
                 </div>
```

Comment:
Breaking on here can make this more readable:
<bl-start-task-picker
    [formControl]="startTaskPicker.nestedValue"
    [userAccounts]="form.value.userAccounts"
    [virtualMachineConfiguration]="form.value.os.virtualMachineConfiguration">
</bl-start-task-picker>